tests/test_nibblestorage.py

```diff
@@ -124,8 +124,7 @@ def test_read_write():
     sct.save(fname)
 
     # on purpose choose parameters that are different from sct
-    sct2 = SmallCounttable(3, 1e4, 2)
-    sct2.load(fname)
+    sct2 = SmallCounttable.load(fname)
```

Comment:
This constructor makes much more sense than the two-step allocate/load we previously had to do.